DQM/Integration/python/clients/hcal_dqm_sourceclient-live_cfg.py

```diff
@@ -243,6 +254,10 @@
 )
 process.options.wantSummary = True
 
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(-1)
+    )
```

Comment:
```suggestion
```

in live mode the number of events processed is regulated elsewhere:

https://github.com/cms-sw/cmssw/blob/f97584b622ab5e783f4fe2a50611c1e269b65629/DQM/Integration/python/config/inputsource_cfi.py#L111-L125

Reply:
Removed in most recent commit